README.md

```diff
@@ -1,13 +1,24 @@
-Beautiful text chat for your community and website.
+## Scrollback, where communities hang out
 
-# Installation
+Scrollback gives you beautiful text chat for your community and website. [Try it out](https://scrollback.io/me)! 
 
-Run this command in terminal:
+You can even [embed Scrollback room](https://github.com/scrollback/scrollback/wiki/Basic-Usage#embed-scrollback-room) on your webpage/blog.
 
-    wget https://raw.githubusercontent.com/scrollback/scrollback/master/tools/install.sh -O install.sh && chmod +x install.sh && ./install.sh
+## Install
 
-To run Scrollback run:
+Scrollback is hosted Free Software but if you wanted to run it on your computer, you can! To install Scrollback locally:
 
```

Comment:
Please change to:
Scrollback provides a free-to-use hosted server for communities, at [scrollback.io](https://scrollback.io/me). If you’re planning run a Scrollback community, this is almost certainly what you want to use.

However, if you want to run Scrollback on your own Linux or OSX machine, run this command: